server/mix.exs

```diff
@@ -45,6 +45,10 @@ defmodule BuffServer.MixProject do
       {:argon2_elixir, "~> 2.0"},
       {:comeonin, "~> 5.1"},
       {:joken, "~> 2.0"},
+      {:ex_aws, "~> 2.0"},
+      {:ex_aws_s3, "~> 2.0"},
+      {:poison, "~> 3.0"},
```

Comment:
no chance they work with Jason? just having two json encoders is.. buh..

Reply:
let me try